proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -156,84 +156,84 @@ message PayloadOrganizationMemberRemoved {
   buf.alpha.registry.v1alpha1.OrganizationRole member_role = 3;
 }
 
-message PayloadRepositoryCreated {
-  // owner_id is the id of the owner of the repository.
+message PayloadModuleCreated {
+  // owner_id is the id of the owner of the module.
   string owner_id = 1;
-  // owner_name is the name of the owner of the repository.
+  // owner_name is the name of the owner of the module.
   string owner_name = 2;
 }
 
-message PayloadRepositoryDeleted {
-  // owner_id is the id of the owner of the repository.
+message PayloadModuleDeleted {
+  // owner_id is the id of the owner of the module.
   string owner_id = 1;
-  // owner_name is the name of the owner of the repository.
+  // owner_name is the name of the owner of the module.
   string owner_name = 2;
 }
 
-message PayloadRepositoryCommitPushed {
-  // owner_id is the id of the owner of the repository.
+message PayloadModuleCommitPushed {
+  // owner_id is the id of the owner of the module.
   string owner_id = 1;
-  // owner_name is the name of the owner of the repository.
+  // owner_name is the name of the owner of the module.
   string owner_name = 2;
-  // repository_id is the id of the repository within which the commit was created.
-  string repository_id = 3;
-  // repository_name is the name of the repository within which the commit was created.
-  string repository_name = 4;
+  // module_id is the id of the module within which the commit was created.
+  string module_id = 3;
+  // module_name is the name of the module within which the commit was created.
+  string module_name = 4;
   // tags are the tags included in the push. Optional.
   repeated string tags = 5;
   // draft_name is the name of the draft. Optional.
   string draft_name = 6;
 }
 
-message PayloadRepositoryContributorAdded {
-  // owner_id is the id of the owner of the repository.
+message PayloadModuleContributorAdded {
+  // owner_id is the id of the owner of the module.
   string owner_id = 1;
-  // owner_name is the name of the owner of the repository.
+  // owner_name is the name of the owner of the module.
   string owner_name = 2;
-  // repository_id is the id of the repository with the new contributor.
-  string repository_id = 3;
-  // repository_name is the name of the repository with the new contributor.
-  string repository_name = 4;
-  // contributor_role is the role granted to the contributor added to the repository.
+  // module_id is the id of the module with the new contributor.
+  string module_id = 3;
+  // module_name is the name of the module with the new contributor.
+  string module_name = 4;
+  // contributor_role is the role granted to the contributor added to the module.
   buf.alpha.registry.v1alpha1.RepositoryRole contributor_role = 5;
```

Comment:
I didn't change the name of `buf.alpha.registry.v1alpha1.RepositoryRole` since it's more widely used and part of our "v1 supported" APIs; figured renaming it would be part of a larger effort (if at all possible)

Reply:
How do I determine if something is part of our supported APIs? I would think as part of the rename, everything gets renamed?

Reply:
> How do I determine if something is part of our supported APIs?

I believe everything in the `v1alpha1` registry API is being considered "v1", meaning we can't make backwards incompatible changes. I would need to defer to @smallsamantha for more detail though.

> I would think as part of the rename, everything gets renamed?

Agreed, but I think it's safer and more tightly scoped to modify _just_ the messages in the audit APIs.

Reply:
`v1alpha` is still alpha and subject to changes, BSR (UI and API) are not yet at v1, although breaking changes not much desirable, this module+snapshot thing is being changes almost everywhere.

Reply:
I found this:
> - APIs, protos, generated client code are not changed and are still using repository
> - Anything that requires objects directly from the API layer uses the term repository  (PrintRepositoryCommit function taking a RepositoryCommit)

From https://linear.app/bufbuild/issue/TCN-507/rename-commit-to-snapshot-repository-to-module

So it looks like we don't need to rename this existing stuff, just audit logging. I won't block this PR on it, but could you confirm with @joshcarp?

Reply:
> So it looks like we don't need to rename this existing stuff, just audit logging.

This was my takeaway from a [conversation in yesterday's team recap](https://www.notion.so/bufbuild/BSR-Team-Recap-a0d0c15b347b47c58aa38e86027da7fc#4e4d6b8a48864a6a8d43b7124edc5afd) cc @doriable 

> I won't block this PR on it, but could you confirm with @joshcarp?

sure thing!

Reply:
API changes were already attempted but deemed too messy to be done throughout everything.
This set of changes are meant to just be the surface level with only the frontend/buf cli being changed.
It was decided in a meeting with @renahlee and @SyrieBianco and I that a complete refactor has already been attempted and was deemed too messy so stuff like the API and the buf.yaml files should remain as `repository` and `commit` at the moment.

Reply:
Although obviously if the API is getting promoted to stable soon then we should definitely talk about renaming it before that happens, but that's not the case with the cli tool as everything referencing `repository` and `commit` are still in beta